engine/testabilities/fixture_engine.go

```diff
@@ -117,8 +113,13 @@ func (f *engineFixture) ConfigForTests(opts ...ConfigOpts) *config.AppConfig {
 	return configuration
 }
 
-// initDbConnection creates a new connection that will be used as connection for engine
-func (f *engineFixture) initDbConnection() {
+// prepareDBConfigForTests creates a new connection that will be used as connection for engine
+func (f *engineFixture) prepareDBConfigForTests() {
+	if f.config.Db.Datastore.Engine != datastore.SQLite {
+		panic("Other datastore engines are not supported in tests (yet)")
```

Comment:
I strongly recommend using `t.Fatal` or `t.Fatal`f from the built-in `testing` package. These functions signal test failures in a controlled manner, ensuring the framework handles them internally (e.g., logging the current state and terminating the test). Unlike calling panic directly, they avoid unintended side effects, such as disrupting other tests or goroutines, providing a cleaner and more predictable testing process.
 
 
```
if f.config.Db.Datastore.Engine != datastore.SQLite {
		f.t.Fatal("Other datastore engines are not supported in tests (yet)")
	}
```

Reply:
I'm changing it to require.Equal